python-frontend/src/test/java/org/sonar/python/tree/PythonTreeMakerTest.java

```diff
@@ -2084,6 +2083,24 @@ public void string_interpolation_nested_expressions_in_format_specifier() {
     assertThat(formatSpecifier.formatExpressions().get(1).expression().is(Kind.MULTIPLICATION)).isTrue();
   }
 
+  @Test
+  public void string_multiple_expressions() {
```

Comment:
To be more precise, it yields a tuple instead of multiple expressions

Reply:
done